Source/WebCore/Modules/modern-media-controls/controls/text-tracks.css

```diff
@@ -76,6 +77,20 @@ video[controls]::-webkit-media-text-track-container.visible-controls-bar {
     display: inline-block;
 }
 
+::-webkit-media-text-track-region {
+    position: absolute;
+    writing-mode: horizontal-tb;
+    background: rgna(0,0,0,0.8);
```

Comment:
should this be `rgba` instead of `rgna`?

Reply:
Yes!